app/uk/gov/hmrc/tai/config/ApplicationConfig.scala

```diff
@@ -27,18 +27,22 @@ class ApplicationConfig extends DefaultServicesConfig {
   lazy val citizenAuthHost = fetchUrl("citizen-auth")
   lazy val companyAuthUrl = fetchUrl("company-auth")
   lazy val urlEncode = helper.urlEncode(_: String)
-  lazy val incomeTaxFormPartialLinkUrl = s"${fetchUrl("iform-partial")}/forms/personal-tax/income-tax/catalogue"
+  lazy val incomeTaxFormPartialLinkUrl =
+    s"${fetchUrl("dfs-digital-forms-frontend")}/digital-forms/forms/personal-tax/income-tax/catalogue"
   lazy val incomeFromEmploymentPensionPartialLinkUrl =
-    s"${fetchUrl("iform-partial")}/forms/form/tell-us-about-income-from-employment-or-pension/guide"
+    s"${fetchUrl("dfs-digital-forms-frontend")}/digital-forms/form/tell-us-about-income-from-employment-or-pension/draft/guide"
   lazy val incomeFromEmploymentPensionLinkUrl =
-    s"${fetchUrl("dfs-frontend")}/forms/form/tell-us-about-income-from-employment-or-pension/guide"
-  lazy val companyBenefitsLinkUrl = s"${fetchUrl("dfs-frontend")}/forms/form/tell-us-about-company-benefits/guide"
+    s"${fetchUrl("dfs-digital-forms-frontend")}/digital-forms/form/tell-us-about-income-from-employment-or-pension/draft/guide"
```

Comment:
Can you explain why these two are identical when one is intended to load a partial whilst the other is a redirect?

Reply:
The application.conf entries for ```dfs-frontend``` and ```iform-partial``` were identical as far as I could tell. I'll put restore the partial and update its config so that it targets the new frontend.